### ISSUE TITLE
Homepage section base on tracking app??

### ISSUE THREAD

**Issue by MrConqueror** [LagradOst/CloudStream-3#902]
Home is based on the site we scrap. May i ask Why wouldn't you base it with tracking apps?

**Comment by RohitKing2001** [comment]
yeah! That will be better for recommendations

**Comment by Jhawahar** [comment]
Yep we can have come home ui for all movied shows and anime and cartoon with tracking and details of that show

**Comment by Jasonsmith236** [comment]
That would be easier to navigate

**Comment by MrConqueror** [comment]
@RohitKing2001 @Jhawahar @Jasonsmith236 Yep, i believe cs2 was based on tracking apps. Just like Saikou is based on Anilist. so it's easy to navigate animes with a lot of sequel, then we can also have recommendations, Filter by Genre, and Characters Info. This app has a lot of movies or anime to scrap the only problems is the home section example Gogoanime as a source then homesection is shows what Gogoanime site is showing n their site. And personally i don't like it, Recently dub, Recently Sub, Chinese anime ah fuck it. 
![Screenshot_2022-04-12-09-18-10-84](https://user-images.githubusercontent.com/102837820/162859331-23f27aba-8971-417b-a7a2-1ba99ee9086e.jpg)
⬆️ This is what we want as home section, for movies. and for Anime ofcourse it's either Anilist or Mal etc.

**Comment by Halo39** [comment]
@MrConqueror can I get the link of the app you attached the screenshot of? It looks good I want to try it.
